lib/matplotlib/widgets.py

```diff
@@ -155,6 +156,26 @@ def _set_cursor(self, cursor):
         """Update the canvas cursor."""
         self.ax.get_figure(root=True).canvas.set_cursor(cursor)
 
+    def _save_blit_background(self, background):
+        """
+        Save a blit background.
+
+        The background is stored on the canvas in a uniquely identifiable way.
+        It should be read back via `._load_blit_background`. Be prepared that
+        some events may invalidate the background, in which case
+        `._load_blit_background` will return None.
+
+        This currently allows at most one background per widget, which is
+        good enough for all existing widgets.
+        """
+        if self._blit_background_id is None:
+            self._blit_background_id = self.canvas._get_blit_background_id()
+        self.canvas._blit_backgrounds[self._blit_background_id] = background
```

Comment:
A second thought:

In practice we only have one background per widget, but because the current API is scoped to a bounding box a widget _could_ have more than one.  Do we want to deal with that complexity here or kick that down the road?  This is coupled to Antony's suggestion to standardized the return type.

Reply:
I'd say let's start simple and only do as much as needed to get the current use cases sorted. I've intentionally made everything private so that we have the freedom to iterate on the API.